Resources/Textures/Objects/light_tube.rsi/meta.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
```

Comment:
You should add ```"license"``` and ```"copyright"```, like so:
https://github.com/space-wizards/space-station-14-content/blob/74541e23a46a827d55de91943b9595f2c05d7a4e/Resources/Textures/Mob/human.rsi/meta.json#L7
https://github.com/space-wizards/space-station-14-content/blob/74541e23a46a827d55de91943b9595f2c05d7a4e/Resources/Textures/Mob/human.rsi/meta.json#L8

Reply:
Done!